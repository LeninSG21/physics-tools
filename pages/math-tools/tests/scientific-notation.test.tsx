import { render } from '@testing-library/react';

import ScientificNotation from '../scientific-notation';

describe('<ScientificNotation tests>', () => {
  it.skip('should match snapshot', () => {
    // arrange
    const { container } = render(<ScientificNotation />);

    // assert
    expect(container).toMatchSnapshot();
  });
});
