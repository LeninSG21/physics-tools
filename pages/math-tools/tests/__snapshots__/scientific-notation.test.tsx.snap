// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScientificNotation tests> should match snapshot 1`] = `
<div>
  <div />
</div>
`;
